### PR TITLE
Docker changes to work with python3

### LINKED ISSUE #2: Update hepdata-converter-docker to work with python3 and hepdata-validator 0.2
Once HEPData/hepdata-validator#5 is complete, the hepdata-converter docker image should be updated to use the latest version.

I suggest we should also pin the versions in requirements.txt so that we know which versions of packages were used to build the docker image.

### PR DESCRIPTION
**Pull request by alisonrclarke**
Fixes #2 

Switched to use a conda image from root so that root works with python3.

### CODE REVIEW

**Comment by alisonrclarke** [comment]
The 1.8 version of YODA doesn't work correctly for scatter3ds: see https://gitlab.com/hepcedar/yoda/issues/5

I've made a merge request at https://gitlab.com/hepcedar/yoda/-/merge_requests/14 with a suggested fix.

Once those changes have been merged (or it's fixed elsewhere), we need to decide whether to build a docker image from an unreleased version of YODA or wait for a new release.

**Comment by alisonrclarke** [comment]
https://gitlab.com/hepcedar/yoda/-/merge_requests/14 has now been merged. I've asked when they might be cutting a new release to help us decide whether to release a new docker image using the unreleased version or wait for the latest release.

**Review by GraemeWatt — CHANGES_REQUESTED** [review]
Looks good, I'll merge and tag a new version 0.2.0.  Can you please first bump the `hepdata-validator` version to 0.2.2 in `requirements.txt`?  I tried to push the change to your branch myself, but I don't have permissions.

**Pull request merged**
